Plugins/Reqnroll.CustomPlugin/README.md

```diff
@@ -0,0 +1,9 @@
+﻿Reqnroll.CustomPlugin provides a sample custom plugin for Reqnroll.
+
+This package serves as a template and example for creating custom plugins that extend Reqnroll functionality. It demonstrates how to create custom generators, transformations, and other extensions.
+
+## Support
+
+- [Reqnroll Documentation](https://docs.reqnroll.net/)
```

Comment:
link to https://docs.reqnroll.net/latest/extend/plugins.html#generator-plugins instead